gitian-building.md

```diff
@@ -123,6 +123,7 @@ You may have other .assert files as well (e.g. `signed` ones), in which case you
 This will create the `.sig` files that can be committed together with the `.assert` files to assert your
 Gitian build.
 
- `./gitian-build.sh --detach-sign --nocommit -s satoshi 0.16.0rc1`
+
+ `./gitian-build.sh --detach-sign -s satoshi 0.16.0rc1 --nocommit`
```

Comment:
Independent of this fix, the `s` should be upper case aka `--signer`, because `-s` means `--sign`. And the `--nocommit` is ignored, I think because that option does not exist and no options are parsed after the first non-option (0.16.0rc1).

Reply:
On master `--no-commit` is still in the help, it sets `commitFiles=false` and afaik that does work as advertised.

`-s` should be making signed binaries. `--sign` is a confusing alias, and should probably be `--signed-binaries`. 

`--signer` doesn't exist, it just assumes the first argument is the signer. So `-s` doesn't refer to `satoshi`.

It's pretty confusing...

Reply:
Note the difference between `--nocommit` and `--no-commit` (dashed)